### PR TITLE
Update Dutch & HTML entity

### PR DESCRIPTION
**Pull request by Eitot**
Grammar and style

### CODE REVIEW

File: _layouts/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <link rel="stylesheet" href="./css/pygments.css" type="text/css" media="screen">
     <link rel="stylesheet" href="./css/fonts/chunkfive/chunkfive.css" type="text/css" media="screen">
     {% if page.font-family %}
-    <link href="//fonts.googleapis.com/css?family=PT+Serif:700&subset=latin,cyrillic" rel="stylesheet" type="text/css">
+    <link href="//fonts.googleapis.com/css?family=PT+Serif:700&amp;subset=latin,cyrillic" rel="stylesheet" type="text/css">
```
Comment (bfontaine) [inline _layouts/index.html]: This should be `&`, not `&amp;`.

**Comment by mikemcquaid** [comment]
Can you rebase this on `master` and then I'll merge? Sorry for the delay!

**Comment by Eitot** [comment]
@mikemcquaid: Done.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**